explore/src/main/scala/explore/HomeComponent.scala

```diff
@@ -57,76 +58,115 @@ object HomeComponent {
+  import cats.effect.SyncIO
+  import crystal.react.StreamRendererMod
+  import explore.observationtree.TargetTreeTest
 
-              <.div(
-                GPPStyles.TreeRGL,
-                Resizable(
-                  axis = Axis.X,
-                  width = treeWidth,
-                  height = Option(s.height).getOrElse(0),
-                  minConstraints = (200: JsNumber, 0: JsNumber),
-                  maxConstraints = (s.width.toDouble / 2: JsNumber, 0: JsNumber),
-                  onResize = treeResize,
-                  resizeHandles = List(ResizeHandleAxis.East),
-                  content = tree
-                ),
-                <.div(^.width := coreWidth.px, ^.left := treeWidth.px, GPPStyles.RGLBody)(
-                  ResponsiveReactGridLayout(
-                    width = coreWidth,
-                    margin = (5: JsNumber, 5: JsNumber),
-                    containerPadding = (5: JsNumber, 5: JsNumber),
-                    rowHeight = 30,
-                    draggableHandle = ".tileTitle",
-                    useCSSTransforms =
-                      false, // Not ideal, but fixes flicker on first update (0.18.3).
-                    // onLayoutChange = (a, b) => Callback.log(a.toString) *> Callback.log(b.toString),
-                    layouts = layouts
-                  )(
-                    <.div(
-                      ^.key := "conditions",
-                      ^.cls := "tile",
-                      Tile("Conditions")(
-                        ConditionsPanel(obsId, conditions)
-                      )
-                    ),
-                    <.div(
-                      ^.key := "target",
-                      ^.cls := "tile",
-                      Tile("Target Position")(
-                        TargetEditor(obsId, props.zoomL(RootModel.target), conditions.get.some)
+  import AppCtx._
+
+  val TreeComponent =
+    AppCtx
+      .withCtx { implicit ctx =>
+        val obsRef =
+          SignallingRef
+            .in[SyncIO, IO, List[ExploreObservation]](TargetTreeTest.observations)
+            .unsafeRunSync()
+
+        StreamRendererMod.build(obsRef.discrete)
+      }
+  // END DEMO PURPOSES
+
+  class Backend($ : BackendScope[Props, State]) {
+    private val targetEditorRef = Ref.toScalaComponent(TargetEditor.component)
+
+    def render(props: Props, state: State) = {
+      val obsId = Observation
+        .Id(ProgramId.Science.fromString.getOption("GS-2020A-DS-1").get, Index.One)
+
+      val treeResize = (_: ReactEvent, d: ResizeCallbackData) => $.setState(State(d.size.width))
+
+      conditionsSubscription(obsId) { conditions =>
+        <.div(
+          GPPStyles.RGLArea,
+          SizeMe() { s =>
+            val treeWidth = state.treeWidth.toDouble
+            val coreWidth = s.width.toDouble - treeWidth
+
+            // Tree area
+            val tree =
+              <.div(^.width := treeWidth.px, GPPStyles.Tree)(
+                <.div(GPPStyles.TreeBodyOuter)(
```

Comment:
You may not need it here but the reason for the tree inner element was to leave some space for the scroll bar. We can iterate on that later on

Reply:
Ah, sorry, I misunderstood that, I thought it was only there for the example.